spring-kafka/src/test/java/org/springframework/kafka/listener/adapter/ConvertingAndDelegatingMessageListenerAdapterTest.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2016-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.kafka.listener.adapter;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.junit.jupiter.api.Test;
+
+import org.springframework.kafka.listener.MessageListener;
+import org.springframework.messaging.converter.MappingJackson2MessageConverter;
+import org.springframework.messaging.converter.MessageConversionException;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+/**
+ * @author Adrian Chlebosz
+ * @since 3.0.0
+ *
+ */
+class ConvertingAndDelegatingMessageListenerAdapterTest {
```

Comment:
We prefer to name test classes as `*Tests`.

Reply:
Solved in [603202f](https://github.com/spring-projects/spring-kafka/pull/2165/commits/603202fa68d5a5d0115aceb844f80ad3829d4997)